owlbot.py

```diff
@@ -170,6 +170,7 @@
     os.chdir("protos")
     os.unlink('protos.js')
     os.unlink('protos.d.ts')
+    os.unlink('protos.json')
```

Comment:
This should now delete protos.json. I wonder if you still have it in your local repo?